Emby.Server.Implementations/Emby.Server.Implementations.csproj

```diff
@@ -73,6 +73,12 @@
     <EmbeddedResource Include="Localization\countries.json" />
     <EmbeddedResource Include="Localization\Core\*.json" />
     <EmbeddedResource Include="Localization\Ratings\*.csv" />
+    <EmbeddedResource Include="Plugins\blank.png" />
+    <EmbeddedResource Include="Plugins\Superceded.png" />
+    <EmbeddedResource Include="Plugins\Disabled.png" />
+    <EmbeddedResource Include="Plugins\NotSupported.png" />
+    <EmbeddedResource Include="Plugins\Malfunction.png" />
+    <EmbeddedResource Include="Plugins\RestartRequired.png" />
+    <EmbeddedResource Include="Plugins\Active.png" />
```

Comment:
The image files (if anywhere) belong with the client *theme*, they should be purged from git history before merging this PR.
Just return `MediaBrowser.Model.Plugins.PluginStatus` enum name instread.

Reply:
Removed. PluginStatus is return with the plugin info anyway,

This api was originally there to edit the image similar to the way movies etc are edited - but it increased network usage and was too problematic.